### PR TITLE
Added support for 10 templates: Vd,Tabs,Quote,Taxon,Etim-link and more.

### LINKED ISSUE #897: [IT] List of missing templates
```
 !! Missing 'Vd' template support for word 'badie'
 !! Missing 'Tabs' template support for word 'barcaiolo'
 !! Missing 'Quote' template support for word 'caprone'
 !! Missing 'Taxon' template support for word 'castagnola'
 !! Missing 'Etim-link' template support for word 'cervici'
 !! Missing 'Est' template support for word 'concitatamente'
 !! Missing 'Glossa' template support for word 'direttorio'
 !! Missing 'context' template support for word 'polveri fini'
 !! Missing 'Lett' template support for word 'pedestre'
 !! Missing 'Noetim' template support for word 'strettezza'
```

### PR DESCRIPTION
**Pull request by atti84it**
Fixes #897

### CODE REVIEW

File: wikidict/lang/it/__init__.py [hunk 1]
```diff
@@ -38,8 +38,9 @@
 definitions_to_ignore = (
     "{{verb form",
     "{{nome",
-    "{{agg form",
+    #    "{{agg form",
```
Comment (BoboTiG) [inline wikidict/lang/it/__init__.py:41]:
```suggestion
    #  "{{agg form",
```

File: wikidict/lang/it/__init__.py [hunk 2]
```diff
@@ -51,9 +52,34 @@
...
+    "Glossa": "small(term(parts[1]))",
+    # {{la}}
+    "la": "'latino'",
+    # {{Lett|non comune|it}}
+    "Lett": "small(term('letteralmente'))",
+    # {{Nodef|it}}
+    "Nodef": "'-definizione mancante-'",
+    # {{Noetim||it}}
+    "Noetim": "'-etimologia mancante-'",
+    # {{Quote|Non portar unghie troppo lunghe o sozze, dalle narici non ti spunti il pelo, il fiato non ti sia troppo sgradevole: sotto le nari altrui, tu non putire come un caprone|[[q:Ovidio|Ovidio]]}}
```
Comment (BoboTiG) [inline wikidict/lang/it/__init__.py:72]:
```suggestion
    # {{Quote|...}}
```

File: wikidict/lang/it/__init__.py [hunk 3]
```diff
@@ -51,9 +52,34 @@
...
+    "Quote": "'«' + parts[1] + '» ' + term(parts[2])",
+    # {{Tabs|aggrondato|aggrondati|aggrondata|aggrondate}}
+    "Tabs": "'Masc. sing. ' + parts[1] + ', masc. plur. ' + parts[2] + ', fem. sing. ' + parts[3] + ', fem. plur. ' + parts[4]",
```
Comment (BoboTiG) [inline wikidict/lang/it/__init__.py:75]:
```suggestion
    "Tabs": "'Masc. sing. ' + parts[1] + ', masc. plur. ' + parts[2] + ', fem. sing. ' + parts[3] + ', fem. plur. ' + parts[4]",  # noqa
```

**Comment by BoboTiG** [comment]
Could you add tests for those new templates?

**Comment by atti84it** [comment]
> Could you add tests for those new templates?

Sure. This is the next thing I will do when I will have time. Remember, I asked if you preferred small commits or all the things at the same time, and you said the smaller the better, so I uploaded only what I've done until now.

**Comment by BoboTiG** [comment]
By "the smaller the better" I meant small commits ;)
Ideally 1 commit by template is what we do (inclusing the template handling and 1 or more tests).

But take your time, there is no hurry :)

**Comment by BoboTiG** [comment]
Thanks @atti84it !

**Pull request merged**